Lib/test/test_sax.py

```diff
@@ -774,8 +774,8 @@ class StreamReaderWriterXmlgenTest(XmlgenTest, unittest.TestCase):
     fname = support.TESTFN + '-codecs'
 
     def ioclass(self):
-        writer = codecs.open(self.fname, 'w', encoding='ascii',
-                             errors='xmlcharrefreplace', buffering=0)
+        writer = open(self.fname, 'w', encoding='ascii',
```

Comment:
This invalidates the purpose of the test. This is the test for `StreamReaderWriter`.